hw_3/script.js

```diff
@@ -0,0 +1,111 @@
+function checkArgLength(args) {
+    if (args.length !== 2) {
+        throw new Error("Not 2 arguments");
+    }
+}
+
+function checkMinMax(min, max) {
+    if (!Number.isInteger(min) || !Number.isInteger(max)) {
+        throw new Error("arguments are not Integer");
+    }
+    if (min > max) {
+        throw new Error("Min > Max");
```

Comment:
Also here:
```suggestion
        throw new Error("Minimum value (min) is greater than maximum value (max)");
```

Reply:
Changed the error message.
https://github.com/BlackTruth/javascript/blob/813db56a38899f41dcfdc4b332943bde5c271592/hw_3/script.js#L19-L21